chapter_introduction/index_vn.md

```diff
@@ -1759,13 +1759,17 @@ this is sometimes called *offline learning*.
 For supervised learning, the process looks like :numref:`fig_data_collection`.
 -->
 
-*dịch đoạn phía trên*
+Cho tới giờ, chúng ta chưa thảo luận về việc dữ liệu tới từ đâu hoặc chuyện gì thực sự sẽ *xảy ra* khi một mô hình học máy trả về kết quả dự đoán.
+Điều này là do học có giám sát và học không giám sát chưa giải quyết các vấn đề một cách thấu đáo.
+Trong cả hai cách học, chúng ta đưa vào một lượng dữ liệu lớn từ đầu rồi cho mô hình học mà không cho nó trở lại tương tác với môi trường.
+Bởi vì toàn bộ việc học diễn ra khi thuật toán đã được ngắt kết nối khỏi môi trường, đôi khi ta gọi đó là *học ngoại tuyến* (*offline learning*).
+Quá trình này cho học có giám sát được mô tả trong :numref:`fig_data_collection`.
```

Comment:
```suggestion
Quá trình này cho học có giám sát được mô tả trong hình :numref:`fig_data_collection`.
```

Reply:
@ngcthuong từ "Fig" sẽ được tự động thêm vào như [ở đây](https://d2l.ai/chapter_preface/index.html#content-and-structure).

Về sau sẽ có phần Việt hóa code tự động trích dẫn, việc này có lẽ @quangnhat185 sẽ thích.